robosuite/environments/nut_assembly.py

```diff
@@ -191,12 +191,28 @@ def __init__(
         if placement_initializer:
             self.placement_initializer = placement_initializer
         else:
-            self.placement_initializer = UniformRandomPegsSampler(
-                x_range=[-0.15, 0.],
-                y_range=[-0.2, 0.2],
-                z_range=[0.02, 0.10],
+            # treat sampling of each type of nut differently since we require different
+            # sampling ranges for each
+            self.placement_initializer = SequentialCompositeSampler()
```

Comment:
how easy is it to extend this functionality to arena-specific initializations? (e.g: peg locations). Was looking at `SequentialCompositeSampler` and it looks like it can be used pretty flexibly in this way

Reply:
I think it would be straightforward to do so - but we don't have to handle that in this PR. I agree in principle it would be nice to have python files per arena that just append fixed object locations (e.g. pegs) and merge cameras into the xml as well, but no need to do that right now.